.github/workflows/ts-test-e2e.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  workflow_call: {}
+
+defaults:
+  run:
+    working-directory: ./typescript/apps/washboard-ui
+
+jobs:
+  run:
+    name: E2E Tests
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        wash:
+          - version: 0.32.1
+          - version: 0.33.0
+          - version: 0.34.0
```

Comment:
```suggestion
          - version: 0.34.1
```

Reply:
I wonder if we can automate rolling updates to this...